### PR TITLE
Fix voters filter - Closes #4126

### LINKED ISSUE #4126: Filter voters does not work
### Expected behavior

Should filter voters

### Actual behavior

Does not filter properly

### Steps to reproduce

- Go to wallet and open voting tab
- Use voters filter and observe

### Which version(s) does this affect? (Environment, OS, etc...)

2.2.0-linux

Screenshot from 2022-01-31 13-06-30

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?

This PR resolves #4126 

### How was it solved?

- Fix filter function

### How was it tested?

Add unit test

### CODE REVIEW

File: src/components/screens/wallet/delegateProfile/delegateVotesView.js [hunk 1]
```diff
@@ -13,23 +13,25 @@ import styles from './delegateProfile.css';
...
 
   const handleLoadMore = () => {
     voters.loadData({ aggregate: true, offset: voters.meta.count + voters.meta.offset });
   };
 
-  const votersInfo = searchedAddress
-    ? voters.data.votes.filter(v => v.address === searchedAddress)
+  const votersInfo = searchInput
+    ? voters.data.votes.filter(
+      v => v.username.includes(searchInput) || v.address.includes(searchInput),
```
Comment (eniolam1000752) [inline src/components/screens/wallet/delegateProfile/delegateVotesView.js:28]: if `username` or `address` is undefined then the code breaks just as it did when I checked it against testnet

image

```suggestion
      v => (v.username && v.username.includes(searchInput)) || (v.address && v.address.includes(searchInput)),
```

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

**Review by ikem-legend — DISMISSED** [review]

**Review by eniolam1000752 — DISMISSED** [review]
LGTM

**Review by ikem-legend — APPROVED** [review]

**Review by ManuGowda — COMMENTED** [review]
@isalga the PR looks good, to ensure we have no regression in future, can we add tests?

**Review by ManuGowda — APPROVED** [review]
Great job @isalga the tests are 💯 good

**Review by eniolam1000752 — APPROVED** [review]
LGTM

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**